File: Movies/Baskets.cpp
#include "Baskets.hpp"
#include <string>

Baskets::Baskets()
{}

Baskets::~Baskets()
{}

Baskets::Baskets(std::string name, double price)
{
	this->name = name;
	this->price = price;
	this->quantity = 1;
}

std::string Baskets::getName()
{
	return this->name;
}

double Baskets::getPrice()
{
	return this->price;
}

int Baskets::addQuantity()
{
	quantity += 1;
	return quantity;
}

int Baskets::getQuantity()
{
	return this->quantity;
}

File: Movies/Python/Baskets.py
from Food import Food

class Baskets(Food):
	def __init__(self, name, price):
		self.name = name
		self.price = price
		self.quantity = 1 
		
	def getName(self):
		return self.name
		
	def getPrice(self):
		return self.price
		
	def addQuantity(self):
		self.quantity += 1
		return self.quantity
		
	def getQuantity(self):
		return self.quantity